packages/common/src/api/tan-query/useAuthorizedApps.ts

```diff
@@ -0,0 +1,29 @@
+import { Id } from '@audius/sdk'
+import { useQuery } from '@tanstack/react-query'
+
+import { useAudiusQueryContext } from '~/audius-query'
+import { ID } from '~/models/Identifiers'
+
+import { QUERY_KEYS } from './queryKeys'
+import { QueryOptions } from './types'
+
+export const useAuthorizedApps = (
+  userId: ID | null | undefined,
```

Comment:
pretty sure this is always `currentUserId`? let's just get this inside the hook with `useCurrentUserId`

Reply:
ah right, forgot to change this 👍